reactions/examples/halo_ps.cpp

```diff
@@ -31,17 +36,18 @@ int main(int argc, char* argv[]) {
 	 //output file name
     const char* output = "ps_f5_z0.dat";
 
-    const char* cstr = "transfers/dgp";
+    const char* cstr = "transfers/Matteo_fr";
 
 // 0: scale factor, 1: omega_total, 2-4: mg param (1e-10 ~ GR for default mg functions ), 5: number of points in halo-mass loop in scol_init , 30 works well.
-double vars[6];
+double vars[7];
 
-    vars[0] = 1.;
+    vars[0] = 1./(1.+1.);
```

Comment:
that could be potentially buggy, no? can't you write it in terms of some redshift variable?

Reply:
Yea I agree. Have edited it.